_episodes/19-access-data.md

```diff
@@ -10,12 +10,12 @@ objectives:
 - "Search public STAC repositories of satellite imagery using Python."
 - "Inspect the search result metadata."
 - "Download (a subset of) the assets available for a satellite scene."
-- "Open the satellite imagery as raster data and setup some raster calculations."
+- "Open the satellite imagery as raster data."
 - "Save the processed satellite data to disk."
```

Comment:
```suggestion
- "Save satellite data to disk."
```

Reply:
This point actually became quite short, so I'll merge it with the previous one!